README.md

```diff
@@ -3,13 +3,14 @@
 My name is Mackenzie Whitfield.  
 I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
 
-# Where I have Lived  
+# Where I have Lived 
 1. San Diego  
 2. San Francisco  
 3. Los Angeles
+4. Chicago
 
 # Education
-I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. The experiences I had and the people I met over those for years were some of the best of my life. One of the *most* memorable was my study abroad in Greece. Below is a link to the study abroad page specific to this program. If you go under Student Testimonials, you will see the one I wrote! 
+I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. One of the *most* memorable experiences I had as an undergraduate was my study abroad in Greece. Below is a link to the study abroad page specific to my program. If you go under Student Testimonials, you will see the one I wrote! 
 [LMU Odyssey Program](https://bellarmine.lmu.edu/moderngreek/beyondtheclassroom/studyabroad)
 
 # Future Plans  
```

Comment:
I also really like the clarity of your homework. Adding one or two pictures to your work could help to make your contribution even more memorable!